packages/runtime/container-runtime/src/dataStoreContext.ts

```diff
@@ -774,8 +775,10 @@ export abstract class FluidDataStoreContext extends TypedEventEmitter<IFluidData
                 callSite,
                 pkg: packagePathToTelemetryProperty(this.pkg),
             }, error);
-
-            throw error;
+            // Throw for all clients except summarizer clients
```

Comment:
This is pretty obvious from the code below :-). Can you add details on why throw for non-summarizer clients only.